include/up-cpp/transport/datamodel/UPayload.h

```diff
@@ -39,124 +39,119 @@ namespace uprotocol::utransport {
         UNDEFINED  /* invalid */
     };
 
+    // The Serialization format for the data stored in UPayload.
+    enum UPayloadFormat {
```

Comment:
Please use the protobuf defined UPayloadFormat and NOT this defined enum so there are no issues with values being out of sync.